SinaiStCatherine/SinaiEt004.xml

```diff
@@ -0,0 +1,894 @@
+                                  
+                                  
+                                <item xml:id="a2">
+                                  <locus from="2r" to="3r"/>
+                                  <desc type="GuestText"><title ref="LIT2457Testam" type="incomplete"></title> 
+                                    A few initial lines 
+                                    are missing. Three last lines of the 
+                                  text are written in the bottom margin of the folium. </desc>
+                                  <q xml:lang="gez"><gap reason="illegible"/>ኦወልድየ፡ በቀዳሚት፡ ሰዓተ፡ 
+                                    መዓልት፡ የዓርግ፡ ጸሎቶሙ፡ ለውሉድየ፡ ኀበ፡ እግዚአብሔር።
+                                    <gap reason="ellipsis" resp="DN"/>አርምሞ፡ ለዕለ፡ ኵሎሙ፡<sic resp="DN"> ሰማያን።</sic>
+                                  </q>  
+                                  <note>Cues for the rubricator are written in the margins.</note>
+                                </item>
+                                  
+                                  <item xml:id="a3">
+                                    <locus from="3r" to="4r"/>
+                                    <desc type="GuestText"><title ref="LIT6801KwellomuMalaekt" type="complete"></title></desc>
+                                    <q xml:lang="gez">ኵሎሙ፡ ሰራዊተ፡ መላእክት፡ መንፈሳዊያን፡ ነደ፡ እሳት፡ እሙንቱ፡ 
+                                      ይጸርኁ፡ ወይብሉ፡ ካልእ፡ ለካልዑ፡ ኪሩቤል፡ ይብሉ፡ ቅዱስ፡ ወሱራፌል፡ ይብሉ፡ እኩት፡ ወሩፋኤል፡ ይብሉ፡ ቡሩክ፡
+                                    </q>  
+                                  </item>
+                                  
+                                  <item xml:id="a4">
+                                    <locus from="172r" to="173r"/>
+                                    <desc type="Unclear">A chronological note or commentary on the Virgin Mary and Christ, written in the upper margin in an old hand, 
+                                      marked by a sign looking 
+                                      like <foreign xml:lang="amh">tamallas</foreign>, but its relation to the main text is not clear. 
+                                    </desc>
+                                    <q xml:lang="gez">ኵሎሙ፡ ሰራዊተ፡ መላእክት፡ መንፈሳዊያን፡ ነደ፡ እሳት፡ እሙንቱ፡ 
+                                      ይጸርኁ፡ ወይብሉ፡ ካልእ፡ ለካልዑ፡ ኪሩቤል፡ ይብሉ፡ ቅዱስ፡ ወሱራፌል፡ ይብሉ፡ እኩት፡ ወሩፋኤል፡ ይብሉ፡ ቡሩክ፡
+                                    </q>  
+                                  </item>
+                                  
+                                  
+                                  <item xml:id="a5">
+                                    <locus target="#179r"/>
+                                    <desc type="CommemorativeNote">A note recording the
+                                      death of <persName ref="PRS13951Petros">the priest Ṗeṭros</persName> 
+                                      (obviously the same person as mentioned in <ref type="mss" corresp="SinaiEt004"/>) that
```

Comment:
If it is obvious, probably "obviously" is not needed in the sentence

Reply:
ok